packages/common/src/rule-engine/operators/index.ts

```diff
@@ -0,0 +1,46 @@
+import { OPERATION } from './enums';
```

Comment:
Circular dependency fix when trying to create rules in a separate file - tried to avoid additional bloat to helpers.ts..